### DISCUSSION TITLE
Remove all falsy values from an object and its nested children | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#79] [Announcements]
# Remove all falsy values from an object and its nested children | Leo's dev blog – stories, insights, and ideas

Recursively removes all falsy values (null, undefined, 0, false) from an object and its nested properties.

https://www.leohuynh.dev/snippets/remove-falsy-values-from-an-object

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
